app/lib/katello/concerns/renderer_extensions.rb

```diff
@@ -0,0 +1,21 @@
+module Katello
+  module Concerns
+    module RendererExtensions
+      extend ActiveSupport::Concern
+
+      included do
+        alias_method_chain :kickstart_attributes, :katello
+      end
+
+      def kickstart_attributes_with_katello
+        kickstart_attributes_without_katello
+
+        if @host.content_view && @host.operatingsystem.is_a?(Redhat) &&
+          !@host.operatingsystem.kickstart_repo(@host).blank?
+
+          @mediapath = @host.operatingsystem.mediumpath(@host)
```

Comment:
well it kind of does.  If the user explicitly sets an install media url, we don't want to override that